packages/@sanity/desk-tool/src/components/TimeAgo.js

```diff
@@ -72,7 +74,7 @@ export default class TimeAgo extends React.PureComponent {
   }
 
   render() {
```

Comment:
Could we refactor this component to a functional component that uses `useTimeAgo` from `@sanity/base/hooks`?